archive/demos/index.html

```diff
@@ -145,9 +145,9 @@ <h2>Samples</h2>
   
 <tr> 
 <td style="width: 60px;"> 
-<div style="display: block; text-align: left;"><a href="dj.html" imageanchor="1"><img src="images/dj.png" border="1"></a></div> 
+<div style="display: block; text-align: left;"><a href="../../dj/" imageanchor="1"><img src="images/dj.png" border="1"></a></div> 
```

Comment:
If the DJ app is now served in the landing page, we can pull it out of the archive. So I am perfectly fine with removing the app from this page.

Reply:
I think it might be useful to have the ex-archived demos still in the archive. If you still like, I can remove DJ and Drum Machine from the archive in a future PR!